### DISCUSSION TITLE
Bump transformers extras_require to support latest spacy_transformers

### DISCUSSION THREAD

**Discussion by ringohoffman** [explosion/spaCy#13024] [Help: Installation]
The latest `transformers` minor version (`4.33.*`) is only supported by [`spacy_transformers==1.3.1`](https://github.com/explosion/spacy-transformers/commit/e66983c387326c79da6348baec21351ace6f21c7), but [`spacy` is currently restricted to `spacy_transformers>=1.1.2,<1.3.0`](https://github.com/explosion/spaCy/blob/b4501db6f8bff096f2a2103f998e81cd2dd1dfa2/setup.cfg#L77). Would `spacy` accept a PR increasing this range to `1.1.2,<1.4.0`?

**Comment by ringohoffman** [comment]
https://github.com/explosion/spaCy/pull/13025

**Comment by shadeMe** [comment]
Let's move the discussion to the PR thread.
